diagnostic_aggregator/CMakeLists.txt

```diff
@@ -38,12 +38,13 @@ add_dependencies(${PROJECT_NAME} diagnostic_msgs_generate_messages_cpp)
 add_executable(aggregator_node src/aggregator_node.cpp)
 target_link_libraries(aggregator_node ${catkin_LIBRARIES}
                                       ${PROJECT_NAME}
+                                      ${Boost_INCLUDE_DIRS}
 )
 
 # Analyzer loader allows other users to test that Analyzers load
 add_executable(analyzer_loader test/analyzer_loader.cpp
                                gtest-1.7.0/gtest-all.cc)
-target_link_libraries(analyzer_loader diagnostic_aggregator)
+target_link_libraries(analyzer_loader ${PROJECT_NAME} ${Boost_INCLUDE_DIRS})
```

Comment:
Why is this linking against `${Boost_INCLUDE_DIRS}` ?

Reply:
If you have undefined references to boost, I suggest you try `${Boost_LIBRARIES}` here instead of `${Boost_INCLUDE_DIRS}`.

Reply:
Damn, it is my mistake here, not on the embedded machine! It was an uncorrect copypaste when I reproduced the changes on local repository by hand... Sorry!.. Of course, LIBRARIES should be here.